### ISSUE TITLE
Support for mongo

### ISSUE THREAD

**Issue by opensourcegeek** [paupino/rust-decimal#623]
Is there a plan to support mongo natively, like how you've got a conversion at driver level to NUMERIC type? Mongo supports decimal 128, wondered if that's something this library could target as well out of the box.

**Comment by paupino** [comment]
It is not currently planned, however it's also not outside the realm of possibility. I'm not familiar with the mongodb write protocol so it'd require a bit of research to see how it serializes the type via the driver. At a quick glance, it could be as simple as leveraging bson and serde... but I'd need to really dig a bit deeper.

I'd be certainly open to contributions if anyone gets a chance also!

**Comment by opensourcegeek** [comment]
Yes, I was thinking it'll be helper functions to serialize/deserialize. I can have a look
